code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -91,15 +89,9 @@
 
 /obj/structure/closet/crate/radiation/New()
 	..()
-	new /obj/item/clothing/suit/radiation(src)
-	new /obj/item/clothing/head/radiation(src)
-	new /obj/item/clothing/suit/radiation(src)
-	new /obj/item/clothing/head/radiation(src)
-	new /obj/item/clothing/suit/radiation(src)
-	new /obj/item/clothing/head/radiation(src)
-	new /obj/item/clothing/suit/radiation(src)
-	new /obj/item/clothing/head/radiation(src)
-
+	for(var/i in 1 to 8)
+		new /obj/item/clothing/suit/radiation(src)

```

Comment:
Problem, you put it as 8 radiation suits, it should be 4 radiation suits and 4 radiation helmets